retrieval_service/datastore/providers/firestore.py

```diff
@@ -105,8 +170,80 @@ async def delete_collections(collection_list: list[AsyncCollectionReference]):
                         "terminal": amenity.terminal,
                         "category": amenity.category,
                         "hour": amenity.hour,
+                        # Firebase does not support datetime.time type
+                        "sunday_start_hour": (
+                            str(amenity.sunday_start_hour)
```

Comment:
Do we want to use Timestamp (https://cloud.google.com/python/docs/reference/firestore/latest/google.cloud.firestore_v1.types.Value)?

Reply:
I think Timestamp always includes Date and Time, and these fields have only time (datetime.time). Thoughts?

Reply:
Ok keeping as strings are fine. I don't think there is any benefit for using a timestamp unless we need easier comparisons.